src/XrdXrootd/XrdXrootdConfig.cc

```diff
@@ -1568,7 +1568,8 @@ int XrdXrootdProtocol::xred(XrdOucStream &Config)
         {"rm",       RD_rm},
         {"rmdir",    RD_rmdir},
         {"stat",     RD_stat},
-        {"trunc",    RD_trunc}
+        {"trunc",    RD_trunc},
+        {"write",    RD_write}
```

Comment:
For consistency the entry here should be "openw" and RD_openw. The position in the table will obviously need change.